test/cluster/test_raft_recovery_stuck.py

```diff
@@ -68,7 +68,7 @@ async def test_recover_stuck_raft_recovery(request, manager: ManagerClient):
     await manager.server_start(srv1.server_id)
 
     logging.info(f"Starting {others}")
-    await asyncio.gather(*(manager.server_start(srv.server_id) for srv in others))
+    await manager.rolling_restart(others)
```

Comment:
@patjed41 just for someone that doesnt know, whats the difference bettwen the two?